dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/event/RumEventSerializerTest.kt

```diff
@@ -54,16 +50,10 @@ internal class RumEventSerializerTest {
     }
 
     @Test
-    fun `𝕄 serialize RUM event 𝕎 serialize() with ResourceEvent`(
-        @Forgery fakeEvent: RumEvent,
-        @Forgery event: ResourceEvent
-    ) {
-        val rumEvent = fakeEvent.copy(event = event)
-
-        val serialized = testedSerializer.serialize(rumEvent)
+    fun `𝕄 serialize RUM event 𝕎 serialize() with ResourceEvent`(@Forgery event: ResourceEvent) {
+        val serialized = testedSerializer.serialize(event)
```

Comment:
btw, this line brought the following question to my mind: is new change compatible with data serialized before it? won't we have any problems deserializing data created before this change?

Reply:
yes it is as we are not changing anything in the serialized data format so we do not need a migration mechanism. Already did the necessary tests for it.